test_util/aws.py

```diff
@@ -258,7 +258,15 @@ def create(cls, stack_name, boto_wrapper, template_url,
             'PrivateAgentInstanceType': private_agent_type,
             'PrivateSubnet': private_subnet}
         stack = boto_wrapper.create_stack(stack_name, template_url, parameters)
-        return cls(stack.stack.stack_name, boto_wrapper), SSH_INFO['coreos']
+        try:
+            os_string = template_url.split('/')[-1].split('.')[-2].split('-')[0]
```

Comment:
nit: longer term I'd much rather build this into a "variant" + template name which we break apart, then just do a lookup per general template url shape. For now this works though.